src/app/desktop/windows/terminal/terminal-states.ts

```diff
@@ -37,13 +37,10 @@ export abstract class CommandTerminalState implements TerminalState {
 
   executeCommand(command: string, args: string[]) {
     command = command.toLowerCase();
+    // Add command to protocol
+    this.protocol.unshift(command);
```

Comment:
```suggestion
    // Add command to protocol
    if (!this.commands[command].hideFromProtocol) {
      this.protocol.unshift(command);
    }
```
As there are some (and will be some more) easter egg commands, it is important to have to possibility of not logging some commands.